wdtk-client/src/test/java/org/wikidata/wdtk/client/ClientConfigurationTest.java

```diff
@@ -85,6 +85,7 @@ public void testDefaultArguments() {
 		assertEquals(config.getFilterLanguages(), null);
 		assertEquals(config.getFilterSiteKeys(), null);
 		assertEquals(config.getFilterProperties(), null);
+		assertEquals(config.getReportFilename(), null);
```

Comment:
General comment: assertEquals should always have the expected value in the first argument, and the tested value in the second. This is wrong in much of our existing code, but we should try to fix it when we touch test methods or create new ones.